asab/metrics/service.py

```diff
@@ -1,6 +1,8 @@
 import configparser
 import logging
 import asyncio
+import os
+import psutil
```

Comment:
This is the external dependency. I would like to try it without it - you can basically read that from `/proc/self` file system AFAIK.